.eslintrc.js

```diff
@@ -115,6 +115,8 @@ module.exports = {
     // JSHint "quotmark"
     'quotes': [2, 'single'],
 
-    'semi': [2, 'always']
+    'semi': [2, 'always'],
+
+    'no-bitwise': [2]
```

Comment:
FWIW `'no-bitwise': 2` is enough here. the array notation is only needed when you want to set options on the rule.